{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "de8b4b58_055de907",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 12
      },
      "lineNbr": 0,
      "author": {
        "id": 1310383
      },
      "writtenOn": "2021-12-02T17:31:31Z",
      "side": 1,
      "message": "Going ahead and submitting, the only remaining failure appears to be unrelated and tracked at b/208666108. Thank you for your time and help reviewing this change :)",
      "revId": "d423da13c7c74cc1ced415a54b7b9a85432ca8d5",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}